src/app/components/charts/BarChart.tsx

```diff
@@ -46,6 +46,7 @@ const BarChartCmp = <T extends object>({
         {cartesianGrid && <CartesianGrid vertical={false} stroke={COLORS.antiFlashWhite3} />}
         {withLabels && (
           <YAxis
+            domain={[0, (dataMax: number) => Math.ceil((dataMax + dataMax * 0.2) / 100) * 100]}
```

Comment:
Do we need extra space on Active Accounts too? yaxis doesn't apply to it

Reply:
From Don: "All, please so TPS, Active Accounts, and the Holders (on the Token page)."